resources/views/scenarios/show.blade.php

```diff
@@ -17,6 +23,36 @@
                 @foreach($scenario->choices as $choice)
                     <li class="list-group-item">
                         <div class="card">
+                            @switch($scenario->id)
+                                @case("3")
+                                    <audio id="audio" src="https://www.fesliyanstudios.com/play-mp3/1727" autoplay="false" ></audio>
+                                @break
+                                @case("4")
+                                    <audio id="audio" src="https://www.fesliyanstudios.com/play-mp3/7541" autoplay="false" ></audio>
+                                @break
+                                @case("6")
+                                    <audio id="audio" src="https://www.youtube.com/watch?v=9zw7A9gpTiE&ab_channel=EccentricSounds" autoplay="false"></audio>
+                                @break
+                                @case("7")
+                                    <audio id="audio" src="https://www.fesliyanstudios.com/play-mp3/7151" autoplay="false" ></audio>
+                                @break
+                                @case("9")
+                                    <audio id="audio" src="https://www.fesliyanstudios.com/play-mp3/5674" autoplay="false" ></audio>
+                                @break
+                                @case("10")
+                                    <audio id="audio" src="https://www.fesliyanstudios.com/play-mp3/4002" autoplay="false" ></audio>
+                                @break
+                                //Werkt niet
+{{--                                @case("13")--}}
+{{--                                    <audio id="audio" src="public/audio/man-stop.mp3" type="audio/mpeg" autoplay="false" ></audio>--}}
+{{--                                @break--}}
+                                @case("14")
+                                    <audio id="audio" src="https://www.fesliyanstudios.com/play-mp3/6526" autoplay="false" ></audio>
+                                @break
+                                @case("17")
```

Comment:
Die hardcoded id's zijn misschien niet zo netjes, waar een oplossing via de database misschien netter had geweest.
Al heb ik het zelf ook gedaan in de if die erna komt haha.
Dus kan er niks van zeggen, vind 'm helemaal prima.